dev/bots/docs.sh

```diff
@@ -21,41 +59,37 @@ if [ -d "$FLUTTER_PUB_CACHE" ]; then
   export PUB_CACHE="${PUB_CACHE:-"$FLUTTER_PUB_CACHE"}"
 fi
 
-# Install dartdoc.
-bin/cache/dart-sdk/bin/pub global activate dartdoc 0.20.2
+# Install and activate dartdoc.
+"$PUB" global activate dartdoc 0.20.2
```

Comment:
We can just use `pub` instead of `$PUB` because we already `export PATH="$FLUTTER_BIN:$DART_BIN:$PATH"`? Same with `flutter`, `dart`, ...